app/static/openapi.yaml

```diff
@@ -517,6 +517,94 @@ paths:
         404:
           $ref: '#/components/responses/NotFound'
 
+  /resources/{id}/upvote:
+    put:
+      summary: Upvote Resource - Increment Number of Upvotes
```

Comment:
nit: I think we can remove the `Increment number of Upvotes` and just make it `Upvote Resource` and `Downvote Resource`